components/datadog/actions/post-metric-data/post-metric-data.mjs

```diff
@@ -66,14 +69,14 @@ export default {
     },
   },
   async run({ $ }) {
-    const params = {
-      metric: this.metric,
-      points: this.convertMetricPoints(this.points),
-    };
-    if (this.host) params.host = this.host;
-    if (this.tags) params.tags = this.tags;
-    if (this.metricType) params.metricType = this.metricType;
-    if (this.interval) params.interval = this.interval;
+    const params = lodash.pickBy(lodash.pick(this, [
```

Comment:
I forgot to mention in my comment above that you can import `lodash/pickBy.js` and `lodash/pick.js` individually to avoid including the entire lodash package.
```js
import pick from "lodash/pick.js";
import pickBy from "lodash/pickBy.js";
```

Reply:
Addressed